src/scss/includes/panels/favorite_panel.scss

```diff
@@ -53,13 +53,21 @@ $MASQ_BANNER_HEIGHT: 93px;
 .favorite_panel__item__title {
   max-width: 230px;
   font-size: 16px;
+  line-height: 22px;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
   color: $primary_text;
   font-weight: bold;
 }
 
+.favorite_panel__item__share,
+.favorite_panel__item__delete {
+  width: 30px;
+  font-size: 16px;
+  color: #59595F;
```

Comment:
Can we define and reuse `grey-semi-darkness` in component.scss ?

Reply:
sure